tests/x11/network/yast2_network_setup.pm

```diff
@@ -21,7 +21,14 @@ use utils;
 
 sub run {
     my $self = shift;
+
+    select_console 'root-console';
+    my $nm_is_active = (script_run("readlink /etc/systemd/system/network.service | grep NetworkManager") == 0);
```

Comment:
Could you please extract it to the separate method to `lib/network_utils.pm`? As we could reuse it at least in yast2_lan and yast2_lan_restart.
Also, we could have checked if having NM matches our expectations, as we don't expect it to be on SLE.